scripts/prepare_sector_network.py

```diff
@@ -4914,9 +5042,32 @@ def add_industry(
     )
 
     # remove today's industrial electricity demand by scaling down total electricity demand
-    for ct in n.buses.country.dropna().unique():
+    if study == "walloon-model":
+      wallon_node = config["run"]["wallon_node"]
+      for ct in n.buses[n.buses["carrier"] == "AC"].index:
+        loads_i = n.loads.index[
+          (n.loads.index == ct) & (n.loads.carrier == "electricity")
+        ]
+        if n.loads_t.p_set[loads_i].empty:
+            continue
+        factor = (
+            1
+            - industrial_demand.loc[loads_i, "current electricity"].sum()
+            / n.loads_t.p_set[loads_i].multiply(n.snapshot_weightings.generators, axis=0).sum().sum()
```

Comment:
in prepared_network, the loads shouldn't be multiplied with the snapshot weightings. This is done automatically during solve. If you do it here, it's accounted twice